portfolio/src/main/webapp/script.js

```diff
@@ -2,9 +2,20 @@
  * Delete from datastore
  */
 async function deleteData() {
-  fetch('/delete-data', {method:'POST'}).then(() => getData());
+  fetch('/delete-data', {method:'POST'}).then(() => {
+      document.getElementById("alert-card").style.display = 'none';
+      getData();
+  });
 }
 
+async function deleteBuffer() {
+  fetch('/checklogin').then((response) => response.text())
+  .then((htmlContent) => {
+    const alertCard = document.getElementById('alert-card');
+    alertCard.innerHTML = htmlContent;
```

Comment:
There are certain situations where it makes sense to return the HTML template from the backend. I think with this particular setup, it would make more sense to return a status from the server, then use that status to decide which element to show.

Reply:
Changed :)